tests/CalendarTest.php

```diff
@@ -115,13 +115,12 @@ public function time_stamps_need_to_have_type_time_stamp(): void
         $this->expectException(\InvalidArgumentException::class);
         $this->expectExceptionMessage('Timestamps should have type TimeStamp.');
 
+
```

Comment:
Here is blank line too many,
I wonder if these tests could/should be removed, now that phpstan will throw on them.